test/test_queries.py

```
@@ -0,0 +1,29 @@
+
+def test_strings_load_embedding_via_json_filename(tmp_path: Path) -> None:
+    embedding_group = EmbeddingGroup(items=[StringEmbedding(query="hello world")])
+
+    df = load_embedding_group_dataframe(embedding_group, pd.DataFrame())
+
+    assert isinstance(df, pd.DataFrame)
+    assert df.shape == (1, 512)
+
+
+def test_frame_load_embedding_via_json_filename(
+    video_path: Path, video_embeddings_path: Path
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)